_rootconf_events/2018.md

```diff
@@ -11,6 +11,57 @@ boxoffice_item_collection: "e2ea67ca-e20a-4f89-80c9-20641e04c13d"
 funnel:
   url: "https://rootconf.talkfunnel.com/2018/"
   open: false
+related_events:
```

Comment:
This is inefficient, there's a better way of doing it. Check how SageMaker workshops have done this.

Reply:
Aah, this is neat! Will update.